test/e2e/helloworld.ts

```diff
@@ -1,5 +1,17 @@
 import t = require("unittest/unittest");
 
+class MyClass {
+  field: string;
+
+  MyClass(someVal: string) { this.field = someVal; }
+
+  getField(): string { return this.field + " world"; }
+}
+
 function main(): void {
   t.test("bigifies text", function() { t.expect("hello".toUpperCase(), t.equals("HELLO")); });
```

Comment:
nit: I don't think the e2e test needs to be additive in terms of test cases or assertions. Just keep making it one bigger, more complex program. So we could replace line 12 rather than adding